nucypher/cli/commands/enrico.py

```diff
@@ -49,33 +60,16 @@ def run(general_config, policy_encrypting_key, dry_run, http_port):
 @EnricoInterface.connect_cli('encrypt_message')
 @group_general_config
 def encrypt(general_config, policy_encrypting_key, message):
-    """
-    Encrypt a message under a given policy public key.
-    """
-
-    ### Setup ###
-    emitter = _setup_emitter(general_config, policy_encrypting_key)
-
+    """Encrypt a message under a given policy public key."""
+    emitter = setup_emitter(general_config=general_config, banner=policy_encrypting_key)
```

Comment:
banner? Also, enrico's banner string takes an argument. Shouldn't it be something like:
```suggestion
    emitter = setup_emitter(general_config=general_config, banner=ENRICO_BANNER.format(policy_encrypting_key))
```